Day1/Java/FizzBuzzDimple.java

```diff
@@ -0,0 +1,24 @@
+import java.util.*;

```

Comment:
If you could add the credits at the top that would be great. Follow the formatting instructions mentioned in the [Contributing guidelines](https://github.com/CodeToExpress/dailycodebase/blob/master/CONTRIBUTING.md#pull-request-process-and-contribution-guidelines) to add the author and the date for your Java source code file.